### PR TITLE
Set up new testing env on opendebates aws

### PR DESCRIPTION
**Pull request by dpoirier**
* Update hostnames mainly
* Swap the 'testing' and 'staging' env names in the config
* Add the db settings to the config, including allowing postgres
  replication from any network address. AWS security groups should
  still keep this secure, but we should tighten this down at some
  point.

### CODE REVIEW

File: fabulaws-config.yml [hunk 1]
```diff
@@ -78,11 +78,10 @@
     production:
     -
     staging:
-    - dualstack.opendebates-staging-lb-1737156337.us-east-1.elb.amazonaws.com
+    -
     testing:
-    - testing.demopenforum.com
-    - opendebates-testing.cakt.us
-    - dualstack.opendebates-testing-lb-1802486946.us-east-1.elb.amazonaws.com
+    - opendebates-testing-lb-476601241.us-east-1.elb.amazonaws.com
```
Comment (tobiasmcnulty) [inline fabulaws-config.yml]: Not an issue for this PR, but before we settle on a production ELB URL and commit it here, let's make sure security groups can be setup in VPC to allow traffic only from cloudflare (and our office IPs).

File: fabulaws-config.yml [hunk 2]
```diff
@@ -196,3 +195,66 @@
     - build-essential
     - stunnel4
     - pgbouncer
+
+  db_settings:
```
Comment (tobiasmcnulty) [inline fabulaws-config.yml]: How did this end up in here? Looks like a duplicate of PR #16, which appears to have been merged already.

**Comment by tobiasmcnulty** [comment]
Other than the two "maybe" issues noted, :shipit: from me.

Will need to make sure everyone knows to swap their AWS creds as soon as this gets merged, and tear down all the old instances in Caktus AWS.

File: fabulaws-config.yml (hunk 2 above)
Comment (dpoirier) [inline fabulaws-config.yml]: It looks like I had more comments here, plus adding 0.0.0.0 to pg_networks.  I've merged from develop and updated the pull so it's more obvious what changes are being made.

**Pull request merged**